shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -630,54 +658,57 @@ ex2A :: CHAINExample
 ex2A = CHAINExample (Slot 10) initStEx2A blockEx2A (Right expectedStEx2A)
 
 
--- | Example 2B - process a block late enough in the epoch in order to create a reward update.
--- The block delegates Alice's and Bob's stake to Alice's pool.
--- Additionally, we split Alice's ADA between a base address and a pointer address.
+-- * Example 2B - process a block late enough in the epoch in order to create a reward update.
 
+-- | The transaction delegates Alice's and Bob's stake to Alice's pool.
+--   Additionally, we split Alice's ADA between a base address and a pointer address.
 txbodyEx2B :: TxBody
-txbodyEx2B = TxBody
-           (Set.fromList [TxIn (txid txbodyEx2A) 0])
-           [ TxOut aliceAddr (Coin 729)
-           , TxOut alicePtrAddr (Coin 9000) ]
-           (fromList [ Delegate $ Delegation aliceSHK (hk alicePool)
-           , Delegate $ Delegation bobSHK (hk alicePool)
-           ])
-           Map.empty
-           (Coin 4)
-           (Slot 90)
-           emptyUpdate
+txbodyEx2B = TxBody {
```

Comment:
Since now I see that default style is to comment on arguments instead of exporting record fields,
I changed these to argument comments.
Is it better?

Reply:
we've been trying to use stylish-haskell with the `.stylish-haskell.yaml` in the repo, but I think we never told you that :)

Reply:
@mgudemann I also noticed that the record fields are _not_ imported by default.
I can add them to imports.

But we need to agree on style.
Do we prefer full record with field declarations instead of constructor application?

@JaredCorduan Most of reformattings are not supported by `stylish` (like vertical alignment).
Here the issue seems to be of double `{` (opening brace)

Reply:
Had to switch to qualified records to make it compile, since `TxBody` is also imported from `MockTypes`. Drop me a line if you find a more elegant way of doing that.